<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QWidget" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <widget class="QGroupBox" name="groupBoxDadosPessoias">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>4</y>
     <width>331</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Dados Pessoais</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutDadoPessoais">
     <item>
      <widget class="QLabel" name="labelNome">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNome"/>
     </item>
     <item>
      <widget class="QLabel" name="labelCpf">
       <property name="text">
        <string>CPF:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCpf">
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelRg">
       <property name="text">
        <string>RG:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRg"/>
     </item>
     <item>
      <widget class="QLabel" name="labelDataNascimento">
       <property name="text">
        <string>Data Nascimento:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDataNascimento">
       <property name="inputMethodHints">
        <set>Qt::ImhDate</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelModalidade">
       <property name="text">
        <string>Modalidade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxModalidade">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Karate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Judo</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOutros">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxOutros">
       <property name="text">
        <string>Outros</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDadosContato">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>464</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Dados de Contato</string>
   </property>
   <widget class="QLabel" name="labelEmail">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>25</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>48</y>
      <width>621</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhEmailCharactersOnly</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>79</y>
      <width>621</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutTelefone">
     <property name="spacing">
      <number>14</number>
     </property>
     <item>
      <widget class="QLabel" name="labelTelefoneCelular">
       <property name="text">
        <string>Telefone Celular:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTelefoneFixo">
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTelefoneFixo">
       <property name="text">
        <string>Telefone Fixo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditTelefoneCelular">
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxEndereco">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>4</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Moradia</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>291</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutCepUf">
     <item>
      <widget class="QLabel" name="labelCep">
       <property name="text">
        <string>CEP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelUf">
       <property name="text">
        <string>UF:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>30</y>
      <width>291</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutEndereco">
     <item>
      <widget class="QLabel" name="labelEndereco">
       <property name="text">
        <string>Endereço:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditEndereco"/>
     </item>
     <item>
      <widget class="QLabel" name="labelCidade">
       <property name="text">
        <string>Cidade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCidade"/>
     </item>
     <item>
      <widget class="QLabel" name="labelBairro">
       <property name="text">
        <string>Bairro:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditBairro"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxExtra">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>244</y>
     <width>311</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutExtra">
     <item>
      <widget class="QLabel" name="labelProfissao">
       <property name="text">
        <string>Profissão:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditProfissao"/>
     </item>
     <item>
      <widget class="QLabel" name="labelPeso">
       <property name="text">
        <string>Peso:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPeso"/>
     </item>
     <item>
      <widget class="QLabel" name="labelCategoria">
       <property name="text">
        <string>Categoria:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCategoria">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>981</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonCadastro">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cadastro</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLimpar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Limpar dados</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonAdicionarFoto">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>530</y>
     <width>354</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adicionar foto</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImagem">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>26</y>
     <width>354</width>
     <height>472</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregando imagem...</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditNome</tabstop>
  <tabstop>lineEditCpf</tabstop>
  <tabstop>lineEditRg</tabstop>
  <tabstop>lineEditDataNascimento</tabstop>
  <tabstop>comboBoxModalidade</tabstop>
  <tabstop>lineEditOutros</tabstop>
  <tabstop>checkBoxOutros</tabstop>
  <tabstop>lineEditEndereco</tabstop>
  <tabstop>lineEditCidade</tabstop>
  <tabstop>lineEditBairro</tabstop>
  <tabstop>lineEditCep</tabstop>
  <tabstop>lineEditUf</tabstop>
  <tabstop>lineEditProfissao</tabstop>
  <tabstop>lineEditPeso</tabstop>
  <tabstop>lineEditCategoria</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditTelefoneFixo</tabstop>
  <tabstop>lineEditTelefoneCelular</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
